### DISCUSSION TITLE
Pcaps not Generating in SOC when triggered in Kibana

### DISCUSSION THREAD

**Discussion by ghost** [Security-Onion-Solutions/securityonion#4197] [Unsupported Versions] [answered]
I have done a fresh install of version 2.3.5 in a distributed setup. Everything seems to work except when I try and generate pcaps from clicking on the ID in Kibana. The job in the SOC just stays queued and never completes. I have the same result from all the sensors. Any suggestions?

**Comment by jertel** [comment]
Review the `/opt/so/log/soc/sensoroni-server.log` log file on the manager, and the `/opt/so/log/sensoroni/sensoroni.log` log files on the sensor(s). Look for errors or warnings that could help you learn more about the problem.

**Reply by ghost** [reply]
I had tailed the suggested log files on the sensor and master while attempting to queue the Pcap job. I do not get any errors or warnings during that process.

**Reply by jertel** [reply]
Is your sensor's hostname using mixed case? Ex: mySensor-001

**Reply by ghost** [reply]
They are not. All are lowercase.

**Reply by jertel** [reply]
Screenshots and log snippets will be helpful to better understand what's going on.

**Reply by kiaraho** [reply]
We are also experiencing a similar issue after upgrading our fleet to `2.3.50`. In our case, we do have sensors with mixed case hostnames, though it was not an issue prior to `2.3.50`. PCAPs are still present in the `/nsm/pcap/` directory on each sensor, we just cannot pull them up in the SOC like we used to. There are no errors in the `sensoroni-server.log` file on the Manager. Only 1 of 6 sensors had a error in its `sensoroni.log` file.
```
timestamp=2021-05-21T04:41:04.409064496Z level=error message="Unable to access path while updating epoch" error="lstat /nsm/pcap/.1621572003369368: no such file or directory" path=/nsm/pcap/.1621572003369368
```

**Comment by dougburks** [comment, marked as answer]
Please upgrade to 2.3.51:
https://blog.securityonion.net/2021/05/security-onion-2351-now-available.html

**Reply by ghost** [reply]
Thank you.

The Upgrade did fix the issue.
